models/gif_pop_psc_exp.cpp

```diff
@@ -553,37 +550,37 @@ nest::gif_pop_psc_exp::update( Time const& origin,
     S_.y0_ = B_.currents_.get_value( lag );
 
     // begin procedure update population, Fig. 12
-    double W_ = 0, X_ = 0, Y_ = 0, Z_ = 0; // line 2
+    double W_ = 0, X_ = 0, Y_ = 0, Z_ = 0; // line 2 of [1]
     S_.theta_hat_ = P_.V_T_star_;          // line 2, initialize theta
 
-    S_.V_m_ = ( S_.V_m_ - P_.E_L_ ) * V_.P22_ + h_tot_; // line 3
+    S_.V_m_ = ( S_.V_m_ - P_.E_L_ ) * V_.P22_ + h_tot_; // line 3 of [1]
 
     // compute free adaptation state
-    for ( uint j = 0; j < P_.tau_sfa_.size(); j++ ) // line 4
+    for ( uint j = 0; j < P_.tau_sfa_.size(); ++j ) // line 4 of [1]
     {
-      V_.g_[ j ] = V_.g_[ j ] * V_.Q30_[ j ]
-        + ( 1. - V_.Q30_[ j ] ) * V_.n_[ V_.k0_ ]
-          / ( static_cast< double >( P_.N_ ) * V_.h_ );           // line 5
+      double g_j_tmp_ = ( 1. - V_.Q30_[ j ] ) * V_.n_[ V_.k0_ ]
+          / ( static_cast< double >( P_.N_ ) * V_.h_ );
+      V_.g_[ j ] = V_.g_[ j ] * V_.Q30_[ j ] + g_j_tmp_; // line 5 of [1]
       S_.theta_hat_ = S_.theta_hat_ + V_.Q30K_[ j ] * V_.g_[ j ]; // line 6
     }
 
     // compute free escape rate
-    double lambda_tld_ = escrate( S_.V_m_ - S_.theta_hat_ ); // line 8
+    double lambda_tld_ = escrate( S_.V_m_ - S_.theta_hat_ ); // line 8 of [1]
     double P_free_ = 1 - std::exp( -0.5 * ( V_.lambda_free_ + lambda_tld_ )
-                           * V_.h_ / 1000. );         // line 9
+                           * V_.h_ / 1000. );         // line 9 of [1]
```

Comment:
This could probably also be `const`. Replacing division by 1000. with multiplication by 0.001 would improve performance. Or could you even redefine `V_.h_` in a way to avoid the scaling here?